content/en/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,390 @@
+With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
+Within a namespace, a Pod/Container can still consume as much CPU and memory as the defined namespace's resource quota.
+there is a concern that one Pod/Container could monopolise its namespace resources.
+LimitRange is a tool to address this concern.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+A limit range, defined by a `LimitRange` object, provides constraints that can :
+- Enforce min/max compute resources usage per Pod/Container in a namespace
+- Enforce min/max storage request per PersistentVolumeClaim in a namespace
+- Enforce a ratio between request and limit for a resource in a namespace
+- Set default request/limit for compute resources in a namespace and automatically inject them to Containers at runtime 
+
+
+Limit Range work like this:
+
+- The administrator creates one `LimitRange` in one namespace.
+- Users create resources (Pods, Containers, PersistentVolumeClaim) in the namespace,
+  the `LimitRanger` admission controller force defaults requests/limits if any for Pods/Container that make no compute resource requirements,
+  tracks usage to ensure it does not exceed  resource min/max defined in any  `LimitRange` present in the namespace.
+- If creating or updating a resource violates a limit  constraint, the request will fail with HTTP
+  status code `403 FORBIDDEN` with a message explaining the constraint that would have been violated.
+- If limit range is activated in a namespace for compute resources like `cpu` and `memory`, users must specify
+  requests or limits for those values; otherwise, the system may reject pod creation. 
+
+Examples of policies that could be created using limit range  are:
+
+- In a 2 node cluster with a capacity of 8 GiB RAM, and 16 cores, constraint Pods in the "testing" namespace to request 100m and not exceeds 500m for CPU , request 200Mi and not exceed 600Mi
+- Define default CPU limits and request to 150m and Memory default request to 300Mi for containers started with no cpu and memory requests in their spec.
+
+
+In the case where the total limits of the namespace  is less than the sum of the limits of the Pods/Containers,
+there may be contention for resources.  This is handled on a first-come-first-served basis.
+
+Neither contention nor changes to limitrange will affect already created resources.
+
+## Enabling Limit Range 
+
+Limit Range support is enabled by default for many Kubernetes distributions.  It is
+enabled when the apiserver `--enable-admission-plugins=` flag has `LimitRanger` as
+one of its arguments.
+
+A limit range is enforced in a particular namespace when there is a
+`LimitRange` object in that namespace.
+
+
+
+## Limiting compute resources at Container level
+
+In the following section, we are going to create a LimitRange acting at Container Level.

```

Comment:
General comment: Avoid using "we", since it's unspecified who "we" is. Suggest rewriting this to something like "The following section highlights how to use limitrange at a container level."